prow/test/integration/prow/config.yaml

```diff
@@ -0,0 +1,9 @@
+sinker:
+  resync_period: 5s
+  max_prowjob_age: 48h
+  max_pod_age: 48h
+  terminated_pod_ttl: 30m
+
+prowjob_namespace: default
+pod_namespace: test-pods
+log_level: debug
```

Comment:
It would be handy to allow dumping the Prow component logs into an output dir (`$ARTIFACTS` in CI) so that we can more easily debug integration test failures.

Reply:
It can also be done with client-go, but it might be easier with `kubectl`. We don't need to stream it though, we can just as some kind of a post step dump the log of all pods in a file that has the pod name or sth like that

Reply:
done

Reply:
Yes I'd expect logs to be dumped with kubectl at the end if $ARTIFACTS is populated (or better yet use a CLI arg/flag).

Reply:
And have verified that dumped logs are under artifacts: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/20451/pull-test-infra-integration/1349801426494164992